src/+datools/+uncertainty/Uncertainty.m

```diff
@@ -0,0 +1,17 @@
+classdef (Abstract) Uncertainty < handle
+
+    properties (Abstract)
+        Mean
+        Covariance
+    end
+
+    methods
+        sample(obj, N);
+        addError(obj, x)
+        asGaussian(obj)
+        asGMM(obj)
```

Comment:
How does asGMM work? does it approximate any distribution with a GMM?

Reply:
yes, that is my goal.